### PR TITLE
Useless branch

### PR DESCRIPTION
**Pull request by kptdobe**

### CODE REVIEW

File: src/logger-coralogix.js [hunk 1]
```diff
@@ -18,6 +18,23 @@ const {
 
 let coralogixLogger = null;
 
+// todo: sanitizing the secrets should be better handled in the logging framework.
+const sanitize = (params) => {
+  const sanitizedParams = { ...params };
```
Comment (tripodsan) [inline src/logger-coralogix.js]: @kptdobe this is not a deep clone. i.e. your are destroying the `__ow_headers.authorization` here.

File: src/logger-coralogix.js [hunk 2]
```diff
@@ -38,14 +55,15 @@ function createCoralogixLogger(config, params) {
     const [, , owPackage] = actionName.split('/');
     const applicationName = CORALOGIX_APPLICATION_NAME || namespace;
     const subsystemName = CORALOGIX_SUBSYSTEM_NAME || owPackage || 'n/a';
+    const sanitizedParams = sanitize(params);
```
Comment (tripodsan) [inline src/logger-coralogix.js]: the params are not logged here at all...and the headers never were, due to a bug. so I would remove it completely.

Comment (kptdobe) [reply src/logger-coralogix.js]: ? I do not understand this comment. 2 lines below, I log the `__ow_headers` and this works for the dispatch action, which in my case (see https://github.com/adobe/helix-dispatch/blob/alex-playground/src/index.js#L77) "propagates" the headers to all the other action calls which also receive the headers and can log them.

File: src/logger-coralogix.js (hunk 1 above)
Comment (kptdobe) [reply src/logger-coralogix.js]: I'll fix it in the other branch, thanks for checking.
I had the `ferrum#deepclone` before but it does not work if `__ow_logger` is pretty (cannot copy Logger object). Thus I had to add more magic. Like here https://github.com/adobe/helix-dispatch/blob/alex-playground/src/index.js#L21-L44

**Comment by tripodsan** [comment]
the problem is, that by modifying the `params.__ow_headers.authorization` you destroy that value for the action. it is probably not a problem, since we never use it directly....

**Pull request merged**